docker/Dockerfile.kubectl

```diff
@@ -6,7 +6,8 @@ ARG BASE_VERSION=latest
 FROM docker.io/istio/base:${BASE_VERSION} as default
```

Comment:
I believe in master we no longer use this image after the deletion of the `installer` directory.